src/network/networkd-network-gperf.gperf

```diff
@@ -119,12 +119,13 @@ DHCPServer.EmitTimezone,                config_parse_bool,
 DHCPServer.Timezone,                    config_parse_timezone,                          0,                             offsetof(Network, dhcp_server_timezone)
 DHCPServer.PoolOffset,                  config_parse_uint32,                            0,                             offsetof(Network, dhcp_server_pool_offset)
 DHCPServer.PoolSize,                    config_parse_uint32,                            0,                             offsetof(Network, dhcp_server_pool_size)
-Bridge.Cost,                            config_parse_unsigned,                          0,                             offsetof(Network, cost)
+Bridge.Cost,                            config_parse_uint32,                            0,                             offsetof(Network, cost)
```

Comment:
uh, this is problematic. You also have to change the type in the Network structure now... The config_parse_xxx() call always should match the type used. Meaning: if there's a variable of type "uint32_t" you should use config_parse_uint32 and if there's one of type "unsigned" you should use config_parse_unsigned... Hence, please fix the type here too...

Yes, on all archs we support uint32_t and unsigned are the same size, but still, we should always use the appropriate types through the entire pipeline